api/migrations/20210525230253-create-reminder.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
```

Comment:
The file name `20210525230253-create-reminder.js` should update to latest date.

Reply:
I was confused in what repo I should run the `npx sequelize-cli model:generate` command, but I have figured it out now!